src/utils/queryKeys.ts

```diff
@@ -155,6 +159,7 @@ export const QUERY_KEYS = {
     scaleCoef?.toString(),
     periodsInFarm,
   ],
+  minWithdrawalFee: ["minWithdrawalFee"],
```

Comment:
jeste `QUERY_KEY_PREFIX`

Reply:
to tam nepotrebujeme, protoze to je konstanta